### ISSUE TITLE
Rename <accessrestrict> and <userestrict> and update their content model

### ISSUE THREAD

**Issue by kerstarno** [SAA-SDT/eas-schemas#60]
## Creator of issue

1. Kerstin Arnold
2. EAD team lead, TS-EAS
3. @kerstarno
4. kerstin.arnold@archivesportaleurope.net

## The issue relates to

- [ ] EAC-CPF schema issue
- [ ] EAC-CPF Tag Library issue
- [x] EAD schema issue
- [x] EAD Tag Library issue
- [ ] Schema issue
- [ ] Tag Library issue
- [ ] Suggestions for all schemas
- [ ] Suggestions for all Tag Libraries
- [ ] Other

## Wanted change/feature

Text: `<accessrestrict>` and `<userestrict>` should be renamed to `<accessConditions>` and `<useConditions>` respectively. Along with the renaming, the following should be considered: 
* Reviewing the definitions for additional clarity about the distinctions between different elements and to aid compatibility with RiC1.0
* Offering fuller examples within the tag library and Best Practice Guide for content other than free text. This will aid software developers and aggregators, as well as individual institutions, in standardising encoding of, e.g., condition types and the dates when restrictions apply

_**Note for working on and testing this issue: When the schema changes are done in development branch, please mark the tasks on the highest levels of the list (printed in bold) by ticking the box. When the changes have been tested successfully, please mark the tasks on the lowest level of the list.**_ 

- [x] Rename `<accessrestrict>` to `<accessConditions>`
- [x] Rename `<userestrict>` to `<useConditions>`

**Comment by kerstarno** [comment]
Following the revised discussion of #24 at the Washington meeting on 24 July 2023, 

- [ ] Add `<dateRange>` as a direct sub-element of `<accessConditions>`
- [ ] Add `<dateRange>` as a direct sub-element of `<useConditions>`

For documentation, ensure that a variety of use cases is covered, e.g. time-bound dates (“until 2100”), ambiguous dates (“until death of donor”), perpetual dates (“forever”).

**Comment by fordmadox** [comment]
Now in https://github.com/SAA-SDT/eas-schemas/tree/issue_60

**Comment by kerstarno** [comment]
Following the EAD sub-team's decision during their September 2023 meeting: 

- [ ] Add `@valueURI`, `@vocabularySource`, and `@vocabularySourceURI` to `<accessConditions>`
- [ ] Add `@valueURI`, `@vocabularySource`, and `@vocabularySourceURI` to `<useConditions>`

**Comment by kerstarno** [comment]
Tested with the XSD and the RNG. It seems that the initial change to add `<dateRange>` was never merged to the development branch, so this is missing from the current version. The name change and the addition of the vocabulary attributes, on the other hand, can be confirmed. 

I deleted the old "issue_60" branch and created a new one with the same name based on the current "development" branch as there've been quite some changes in the last three months. I created a pull request for this (#83), which I hope won't collide with the other pull requests of today. Once this has been merged, this issue will need another round of testing.
